components/com_kunena/site/controller/widget/announcement/display.php

```diff
@@ -45,6 +45,13 @@ protected function before()
 			return false;
 		}
 
+		$view = JRequest::getWord ( 'view', 'default' );
+		$layout = JRequest::getWord ( 'layout', 'default' );
```

Comment:
JInput should used here instead of use JRequest, because JRequest will be replaced by JInput call